content/id/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,214 @@
+Karena itu, Job haruslah _idempotent._
+
+Untuk informasi lanjut mengenai keterbatasan, lihat [CronJob](/docs/concepts/workloads/controllers/cron-Job).
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* {{< include "task-tutorial-prereqs.md" >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Membuat Sebuah CronJob
+
+CronJob  membutuhkan sebuah berkas konfigurasi.
+Ini adalah contoh dari berkas konfigurasi CronJob `.spec` yang akan mencetak waktu sekarang dan pesan hallo setiap menit:
+
+{{< codenew file="application/Job/cronJob.yaml" >}}
+
+Jalankan contoh CronJob menggunakan perintah berikut:
+
+```shell
+kubectl create -f https://k8s.io/examples/application/Job/cronJob.yaml
+```
+Keluaran akan mirip dengan ini:
+
+```
+cronJob.batch/hello created
+```
+
+Atau kamu dapat menggunakan `kubectl run` untuk membuat sebuah CronJob tanpa menulis sebuah konfigurasi yang lengkap:
+
+```shell
+kubectl run hello --schedule="*/1 * * * *" --restart=OnFailure --image=busybox -- /bin/sh -c "date; echo Hallo dari klaster Kubernetes"
+```
+
+Setelah membuat sebuah CronJob, untuk mengecek statusnya kalian dapat menggunakan perintah berikut:
+
+```shell
+kubectl get cronJob hello
+```
+Keluaran akan mirip dengan ini:
+
+```
+NAME    SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+hello   */1 * * * *   False     0        <none>          10s
+```
+
+Seperti yang kamu lihat dari hasil perintah di atas, CronJob belum menjadwalkan atau menjalankan pekerjaan apa pun.
+Mengamati Job yang akan dibuat sekitar satu menit:
+
+```shell
+kubectl get Job --watch
+```
+Keluaran akan mirip dengan ini:
+
+```
+NAME               COMPLETIONS   DURATION   AGE
+hello-4111706356   0/1                      0s
+hello-4111706356   0/1           0s         0s
+hello-4111706356   1/1           5s         5s
+```
+
+Sekarang kamu telah melihat satu Job berjalan yang dijadwalkan oleh "hello" CronJob.
+Kamu dapat berhenti mengamati Job dan melihat CronJob lagi untuk melihat CronJob menjadwalkan sebuah Job:
+
+```shell
+kubectl get cronJob hello
+```
+Keluaran akan mirip dengan ini:
+
+```
+NAME    SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+hello   */1 * * * *   False     0        50s             75s
+```
+
+Kamu dapat melihat bahwa CronJob `hello` berhasil menjadwalkan sebuah Job pada waktu yang ditentukan dalam `LAST SCHEDULE`. Saat ini ada 0 Job yang aktif,  berarti sebuah Job telah selesai atau gagal.
+
+Sekarang, temukan Pod yang dibuat oleh jadwal Job terakhir dan melihat keluaran bawaan dari salah satu Pod
+
+{{< note >}}
+Nama Job dan nama Pod itu berbeda.
+{{< /note >}}
+
+```shell
+# Ganti "hello-4111706356" dengan nama Job di sistem kamu
+pods=$(kubectl get pods --selector=Job-name=hello-4111706356 --output=jsonpath={.items[*].metadata.name})
+```
+Menampilkan log sebuah Pod:
+
+```shell
+kubectl logs $pods
+```
+Keluaran akan mirip dengan ini:
+
+```
+Fri Feb 22 11:02:09 UTC 2019
+Hello dari klaster Kubernetes
```

Comment:
Is that the correct output? It will be English unless the code changed.

Reply:
in secod way how t run cronjob I change the echo valu from cron job but ya if user use example cronjob deployment from k8s it will use English, so its better not to change the content? @sftim

Reply:
I think it makes sense to use the English example manifest, and not change this line.

Reply:
oke I'll update